ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateTransactionValidator.java

```diff
@@ -35,7 +35,9 @@ public PrivateTransactionValidator(final Optional<BigInteger> chainId) {
   }
 
   public ValidationResult<TransactionValidator.TransactionInvalidReason> validate(
```

Comment:
NIT: I believe in most places we'll want `allowFutureNonce = false`. We could overload the `validate()` method with an option without the parameter, that would default it to false.